dts/bindings/clock/nordic,nrf-lfxo.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF low-frequency crystal oscillator
+
+compatible: "nordic,nrf-lfxo"
+
+include: [fixed-clock.yaml]
+
+properties:
+  clock-frequency:
+    const: 32768
+
+  load-capacitors:
+    type: string
+    enum:
+      - "default"
+      - "internal"
+      - "external"
+    required: true
+    description: |
+      Type of load capacitors connected to the crystal. Please note that
+      "default" may not necessarily imply the reset value, as the choice may be
+      adjusted when the device trimming happens during system initialization.
```

Comment:
I realized now that this doesn't contain the latest version of the dt-binding (probably because I rebased incorrectly my PR, sorry). It should be

```
  load-capacitors:
    type: string
    enum:
      - "internal"
      - "external"
    description: |
      Type of load capacitors connected to the crystal. If not specified,
      adjustments may still happen when the device trimming happens during
      system initialization.
```